db/migrate/20231129180553_fix_column_name.rb

```diff
@@ -0,0 +1,8 @@
+class FixColumnName < ActiveRecord::Migration[7.1]
+  def change
+    rename_column :users, :Name, :name
+    rename_column :users, :Photo, :photo
+    rename_column :users, :Bio, :bio
+    rename_column :users, :PostCounter, :posts_counter
+  end
+end
```

Comment:
- [ ] [optional] Kindly consider removing this migration file, the already named columns are in small letters already.